auth-service/src/main/kotlin/uk/gov/communities/delta/auth/services/PasswordTokenService.kt

```diff
@@ -9,7 +9,7 @@ import java.sql.Timestamp
 import java.time.Instant
 import kotlin.time.Duration.Companion.hours
 
-class SetPasswordTokenService(private val dbPool: DbPool, private val timeSource: TimeSource) {
+class PasswordTokenService(private val dbPool: DbPool, private val timeSource: TimeSource) {
```

Comment:
This might be a good place for inheritance, with an abstract `PasswordTokenService`, with `RegistrationSetPasswordTokenService` and `ResetPasswordTokenService` inheriting from it and providing the table name